packages/cli/lib/main.js

```diff
@@ -0,0 +1,37 @@
+import { defineCommand, runMain, showUsage } from 'citty'
+import { colors } from 'consola/utils'
+import path from 'node:path'
+import { URL } from 'node:url'
+import { getPackageJSON } from './utils.js'
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname)
+const { version, description } = getPackageJSON(path.resolve(__dirname, '..'))
```

Comment:
If I remember correctly, you should be able to import `.json` file directly while using node.

Just write `import { version, description }` from '../package.json' and it should works.

Reply:
I am implementing cli based on es module. So the rolldown package must have `"type": "module"`.

And in that case, node throw the following error when importing json:
```sh
TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///path/to/rolldown/packages/rolldown/package.json" needs an import attribute of type "json
```
We must use import assertion as follow:

```js
import pkgJson from '../package.json' assert { type: 'json' }
```